HW_1.py

```diff
@@ -0,0 +1,253 @@
+# Task 1
+import datetime
+
+class User:
+    def __init__(self, user):
+        self.user = user
+
+class Message(User):
+    def __init__(self, user, text):
+        self.text = text
+        self.user = user
+        self.datetime = datetime.datetime.now()
```

Comment:
![image](https://user-images.githubusercontent.com/48160953/225772155-d8285810-7ceb-4562-b0b5-3104c306b2e3.png)

```suggestion
        self.datetime = None
```